testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeOauth2HttpsProxy.java

```diff
@@ -40,9 +42,14 @@ public class EnvSinglenodeOauth2HttpsProxy
     private final ResourceProvider configDir;
 
     @Inject
-    public EnvSinglenodeOauth2HttpsProxy(DockerFiles dockerFiles, PortBinder binder, Standard standard, HydraIdentityProvider hydraIdentityProvider)
+    public EnvSinglenodeOauth2HttpsProxy(
+            DockerFiles dockerFiles,
+            PortBinder binder,
+            Standard standard,
+            HydraIdentityProvider hydraIdentityProvider,
+            HttpProxy httpProxy)
     {
-        super(ImmutableList.of(standard, hydraIdentityProvider));
+        super(standard, hydraIdentityProvider, httpProxy);
```

Comment:
Why did we remove `ImmutableList.of` here?

Reply:
Good catch. 
I guess the EnvironmentProvider already creates immutable copy of arguments so I decided there is no sense for adding one more wrapper.
@Praveen2112 @ssheikin wdyt?

Reply:
```
    protected EnvironmentProvider(EnvironmentExtender... bases)
    {
        this(ImmutableList.copyOf(requireNonNull(bases, "bases is null")));
    }
```

Reply:
Then can we have a prep commit for this? As removing `ImmutableList.of` is not related to the current commit.

Reply:
done